src/utils/swarm-session-id.ts

```diff
@@ -10,30 +10,29 @@ export const SWARM_SESSION_ID_KEY = 'swarm-session-id'
  *
  * @param bzzUrl BZZ URL with arbitrary query parameters, e.g. http://.../1231abcd.../valami.html?swarm-session-id=vmi&smth=5
  * @returns url without the SwarmSessionID
+ * @throws if the sessionId is not found in the URL
  */
-export function removeSwarmSessionIdFromUrl(bzzUrl: string): string {
-  const queryIndex = bzzUrl.indexOf('?')
+export function unpackSwarmSessionIdFromUrl(bzzUrl: string): { sessionId: string; originalUrl: string } {
```

Comment:
Can you please write tests for this function?

Reply:
it is not even necessary so I took out.